spacy_experimental/coref/coref_model.py

```diff
@@ -2,14 +2,14 @@
 
 from thinc.api import Model, chain, get_width
 from thinc.api import PyTorchWrapper, ArgsKwargs
-from thinc.types import Floats2d, Ints2d
+from thinc.types import Floats2d, Floats3d, Ints3d, Ints2d, Ints1d
 from thinc.util import xp2torch, torch2xp
 
 from spacy.tokens import Doc
 
 CorefClusterer: Optional[type]
 try:
-    from .pytorch_coref_model import CorefClusterer
+    from .pytorch_coref_model import MiniClusterer as CorefClusterer
```

Comment:
Let's version the coref model instead I feel right? The current `build_coref_model` would be registered as `spacy_experimental.coref_clusterer.v1` or something like that and the new one would be `.v2`.

Reply:
Yeah I'm not sure what the right call here is. One issue is that the `.v2` has a hard expectation of using the distilled RoBERTa model weights, which is I guess architecturally quite a change from the original?